src/Commands/sql/sanitize/SanitizeUserTableCommands.php

```diff
@@ -80,6 +84,20 @@ public function sanitize($result, CommandData $commandData): void
             $messages[] = dt('User emails sanitized.');
         }
 
+        // Sanitize username.
+        if ($this->isEnabled($options['sanitize-username'])) {
+            [$name_table, $name_column] = $this->getFieldTableDetails('user', 'name');
+            [$uid_table, $uid_column] = $this->getFieldTableDetails('user', 'uid');
+            assert($uid_table === $name_table);
```

Comment:
FWIW to my mind this clashes a little with the existing style where the table and column names have been hardcoded. Not sure if that means it's better to stick with the current style or update the others personally (:

If we _do_ keep it, I think we need to catch any exceptions, we don't want to explode the sanitize command as a whole.